ClinGen/clincoded
# Connects to #1482. Connects to #1553. Connects to #1586.

**Steps to test #1482:**
1. In a Gene-Disease record, add a new **Experimental** evidence.
2. In the **Experimental** evidence curation form, select the **Model Systems** or **Rescue** option from the **Experiment type** dropdown.
3. Then from the **Non-human model organism or cell culture model?** dropdown (or the **Rescue observed in human, non-human model organism, cell culture model, or patient cells?** dropdown for **Rescue**), select the **Non-human model organism** option.
4. Then in the subsequent **Non-human model organism** dropdown, expect to see the following options are spelled as they are shown here:
**Dog (Canis lupus familiaris) 9615
Gerbil (Gerbillinae ) 10045
Rabbit (Oryctolagus cuniculus) 9986
Round worm (Caenorhabditis elegans) 6239
Zebrafish (Danio rerio) 7955**

**Steps to test #1553:**
(The following steps can be used to test with or without an affiliation)
1. In a Gene-Disease record, add a new **Individual** evidence.
2. In the **Individual** evidence curation form, select the **Yes** option from the **Is this Individual a proband** dropdown.
3. Then click the **Copy disease term from Gene-Disease Record** button to associate this Proband with a disease.
4. Scroll down the page, click the **Add ClinVar ID** button to associate this Proband with a variant.
5. Then in the **Individual -- Score Proband** panel, expect to see a link with the variant's preferred title being shown in the yellow alert box.
6. Click on the link and expect to see a new window/tab showing a form in which you can curate a variant's gene impact.

**Steps to test #1586:**
1. Log in with an admin role.
2. Assign the following affiliation IDs to yourself: 10016, 10017, 10018, 10019, 10020.
3. Log out and log in again.
4. Upon being authenticated, expect to see the affiliation modal being shown.
5. And in the affiliation dropdown in the modal, expect to see the 5 new affiliations.
6. Go to ^/users, search/filter for the last name **Collantes**. Expect to see one result being returned the searched user.
